Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.py

```diff
@@ -0,0 +1,613 @@
+                                  url_suffix=url,
+                                  cookies=cookies)
+
+    def _get_cookies(self) -> requests.cookies.RequestsCookieJar:
+        data = {
+            'username': self._auth[0],
+            'password': self._auth[1]
+        }
+        response = self._http_request(method='POST',
+                                      url_suffix='/token/v2/authenticate',
+                                      data=data,
+                                      resp_type='response')
+        return response.cookies
+
+
+def cisco_stealthwatch_query_flows_initialize_command(client: Client, tenant_id: str,
+                                                      start_time: str = None, end_time: str = None,
+                                                      time_range: str = None, limit: str = None,
+                                                      ip_addresses: str = None) -> CommandResults:
+    """Initialize the process of query flows with user params and returns the id of the search
+    to retrieve its results
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant we want to search with
+        start_time (str, optional): The start time of the search. Defaults to None.
+        end_time (str, optional): The end time of the search. Defaults to None.
+        time_range (str, optional): Time range (start and end) of the search. Defaults to None.
+        limit (str, optional): Number of records to return. Defaults to None.
+        ip_addresses (str, optional): The IP addresses to search for. Defaults to None.
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs.
+    """
+    # must provide start_time, time_range or start_time and end_time. else: throw error.
+    if not (start_time or end_time or time_range):
+        raise Exception('Must provide start_time, time_range, or start_time and end_time')
+    if not (time_range or start_time) and end_time:
+        raise Exception('Must provide start_time, time_range, or start_time and end_time')
+
+    # formatting start_time and end_time
+    start_time, end_time = times_handler(start_time, end_time, time_range)
+    if not start_time:
+        raise Exception('Invalid time format. Check: start_time, time_range, and end_time')
+
+    data = remove_empty_elements({
+        "startDateTime": start_time,
+        "endDateTime": end_time,
+        "recordLimit": limit,
+        "subject": {
+            "ipAddresses": {
+                "includes": ip_addresses if isinstance(ip_addresses, list) else [ip_addresses]
+            }
+        }
+    })
+    response = client.initialize_flow_search(tenant_id, data)
+    outputs = dict_safe_get(response, ['data', 'query'])
+    table = tableToMarkdown('Query Flows Initializing Information:', outputs,
+                            headers=['id', 'status', 'percentComplete'], removeNull=True)
```

Comment:
```suggestion
                            headers=['id', 'status', 'percentComplete'], removeNull=True, headerTransform=pascalToSpace)
```
better using this CSP transformer for a better headers view - 
![image](https://user-images.githubusercontent.com/50294648/110023601-05909700-7d36-11eb-8fae-25f4044dacca.png)
True for all commands

Reply:
Understood, but what is CSP?

Reply:
CommonServerPython :) 
That's where we have all of the methods that are commonly used in our scripts + integration, it is imported in every integration.